build.gradle

```diff
@@ -26,7 +26,8 @@ apply plugin: "net.ltgt.apt"
 
 group = "com.google.api"
 archivesBaseName = "gax"
-version = "0.0.28-SNAPSHOT"
+
+project.version = new File("version.txt").text.trim()
```

Comment:
Later, I would like to add tasks to automatically bump the version up, and I am uncomfortable with a build script modifying itself.